src/main/scala/com/workday/esclient/EsAliases.scala

```diff
@@ -177,6 +177,17 @@ case class ReturnedAliases(aliases: Map[String, RoutingInfo])
 case class RoutingInfo(indexRouting: Option[String], searchRouting: Option[String])
 
 /**
+  * Generic trait wrapping Elasticsearch alias information.
+  */
+trait GenericAliasInfo{
```

Comment:
I'd personally prefer to avoid naming things with `Generic`and instead add `1_7` or `5` in the class names for the version-specific implementations.